rbql/csv_utils.py

```diff
@@ -102,3 +102,7 @@ def unquote_fields(fields):
     return [unquote_field(f) for f in fields]
 
 
+def ensure_unicode(text):
```

Comment:
Let's keep all the changes local to rbql_main.py file. This function is very simple (one liner essentially: `text = text if PY3 else text.decode('utf-8')`) but it requires a sys module import.